regex-automata/src/nfa/thompson/nfa.rs

```diff
@@ -1748,6 +1793,18 @@ impl fmt::Debug for State {
             State::Look { ref look, next } => {
                 write!(f, "{:?} => {:?}", look, next.as_usize())
             }
+            State::WriteLookaround { look_idx } => {
+                write!(f, "Write Lookaround: {}", look_idx.as_u32())
+            }
+            State::CheckLookaround { look_idx, positive, next } => {
+                write!(
+                    f,
+                    "Check Lookaround {} is {} => {}",
+                    look_idx.as_u32(),
+                    positive,
+                    next.as_usize()
+                )
```

Comment:
We probably want to adjust to their style?

```suggestion
            State::WriteLookaround { look_idx } => {
                write!(f, "write-look-around({})", look_idx.as_u32())
            }
            State::CheckLookaround { look_idx, positive, next } => {
                write!(
                    f,
                    "check-look-around({} is {}) => {}",
                    look_idx.as_u32(),
                    if positive { "matched" } else { "not matched" },
                    next.as_usize()
                )
```